back-end/services/Accountability.js

```diff
@@ -69,6 +69,20 @@ class Accountability {
             new Api({ rpc, signatureProvider });
     }
 
+    /**
+     * Gets information about an account, with extended response type with common name
+     * @param {string} accountName - account to fetch information
+     * @response {AccountExtended}
+     */
+    async getAccount(accountName) {
+        try {
+            const account = await this.rpc.get_account(accountName)
+            return account
+        } catch(e) {
```

Comment:
various errors are not handled here. please handle them or do not try catch here and let the endpoint handle them (or just fail on unknown errors). the BE will give 404 error status if the account does not exist, I think

Reply:
Ok, I removed the try catch block.